shared/components/_button.scss

```diff
@@ -1,8 +1,33 @@
 // Base
 @mixin nio-button--colorize($background, $foreground) {
+  $button-animation: unique-id();
+  $ghost: transparentize($background, 0.5);
   background-color: $background;
   color: $foreground;
   border-color: $background;
+  &:focus,
```

Comment:
i wonder if `:not(:disabled) { }` if would work here. If so, it would make the `:disabled:hover, :disabled:focus` overrides below unnecessary